src/docs/user/references/DataConnectors.adoc

```diff
@@ -7,63 +7,61 @@ We begin by presenting FTP and SFTP connectors then, URL connector.
 
 ==== FTP and SFTP connectors
 
+
+The FTP and SFTP file connector workflows consists of two tasks:
+
+* Import_from_FTP/SFTP_Server: Downloads files or folders recursively from an FTP/SFTP server to the data space.
+* Export_to_FTP/SFTP_Serve: Uploads files or folders recursively from the data space to an FTP/SFTP server.
```

Comment:
data space => which one, global space or user space ? If it can be both please put data spaces.

Reply:
it can be one of them depending on the data management configuration